channelDiff.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"log"
+	"fmt"
+	"github.com/nlopes/slack"
+	"github.com/google/go-cmp/cmp"
+	"gopkg.in/mgo.v2"
+)
+
+type ChannelAndID struct {
```

Comment:
We don't need to make a new type for this. Use the existing `type Channel` and add an `ID` field to it.

Reply:
Yeah, it didn't feel right making that new type, but, for some reason, it felt wrong to modify the existing type `Channel`. I'll fix it.